src/styles/Company.scss

```diff
@@ -18,39 +14,37 @@ h1 {
   text-align: center;
 }
 
-.company-logo {
+.single-company-logo {
   width: 200px;
 }
 
+.category {
+  padding: 15px;
+}
+
 .company-snapshot {
   text-align: center;
-  background-color: rgba($color: #776c6c, $alpha: 0.75);
+  background-color: $accent;
```

Comment:
using a variable here so we can use the same color throughout the app if we want.